Makefile

```diff
@@ -1,7 +1,7 @@
 REGISTRY         ?= docker.io
 ORG              ?= ansibleplaybookbundle
 TAG              ?= latest
-BROKER_IMAGE     ?= $(REGISTRY)/$(ORG)/ansible-service-broker
+BROKER_IMAGE     ?= $(REGISTRY)/$(ORG)/origin-ansible-service-broker
```

Comment:
I'm wondering if we'll need to make this a variable in case we need to make this `ose-` one day.